content/docs/command-reference/update.md

```diff
@@ -38,6 +40,19 @@ to update an imported artifact to a different revision.
 $ dvc update --rev master
 ```
 
+### Worktree update
+
+When using a
+[worktree](/doc/user-guide/data-management/cloud-versioning#worktree-remotes)
+remote, `dvc update` will update the specified target to match the latest
+version of the corresponding file or directory from the remote storage. If the
+"latest" version of the specified target is a deleted file or an empty
+directory, `dvc update` will fail (in order to avoid potential accidental local
+data loss).
```

Comment:
Is this really about accidental data loss? Couldn't the same be said for overwriting local data? Do we need a rationale here?

Reply:
To clarify: not a blocker.